.gitlab-ci.yml

```diff
@@ -25,7 +25,7 @@ coverage:
     - sed -i 's/#\([^ ]\)/\1/g' pubspec.yaml
     - rm -r example
     - ./scripts/prepare.sh
-    - ./scripts/test.sh
+    - ./scripts/test.sh coverage
```

Comment:
wouldn't `which flutter >/dev/null` be `true` because of the flutter image?
If yes, then the `$1` doesn't get used?

Reply:
correct, but why not /shrug :D